config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -6,10 +6,25 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: kfservices.serving.kubeflow.org
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.uri.external
+    name: uri
+    type: string
+  - JSONPath: .status.conditions[?(@.type==\"READY\")].status
+    name: status
+    type: string
+  - JSONPath: .status.default.traffic
+    name: defaulttraffic
+    type: integer
+  - JSONPath: .status.canary.traffic
+    name: canarytraffic
+    type: integer
   group: serving.kubeflow.org
   names:
     kind: KFService
     plural: kfservices
+    shortNames:
+    - kfservice
```

Comment:
how about kfsvc?

Reply:
I was trying to figure out how to do both but it looks like it's not possible with kubebuilder right now. Given the choice, I chose kfservice as it's our actual name :)

Reply:
'kfservice' may simplify as well, given we already have kfserving, kfservice.... so kfsvc maybe introduce another term